tripal/src/TripalDBX/TripalDbxSchema.php

```diff
@@ -931,12 +932,12 @@ public function getTableDef(string $table, array $parameters) :array {
       if (!$table_def && 'tripal' == $source) {
         $valid_source = TRUE;
         $table_def = [];
-        $query = $this->connection->select('tripal_custom_tables','ct');
+        $query = $this->connection->select('tripal_custom_tables', 'ct');
         $query->fields('ct', ['schema']);
         $query->condition('ct.table_name', $table);
         $schema = $query->execute()->fetchField();
         if ($schema) {
-          $table_def = unserialize($schema);
+          $table_def = unserialize($schema, ['allowed_classes' => FALSE]);
```

Comment:
Do we want to add a comment describing why we made this functional change? Just in case we forget later 🤪 🙈